src/App.js

```diff
@@ -67,59 +73,153 @@ class App extends React.Component {
     }
   };
 
+  recordLastNum = (result) => {
+    const regex = /[-+/*]{0,}[0-9]{1,}[.]{0,1}[0-9]*/g;
+    let results = result.match(regex) || [];
+    if (results.length === 0) {
+      return "";
+    }
+
+    //We need to take the last one
+    let lastNum = results[results.length - 1];
+    console.log();
+
+    // If there is no operator -> we return the value
+    if (!"-+/*".includes(lastNum[0])) {
+      return lastNum;
+    }
+    this.setState({
+      operate: lastNum[0],
+    });
+    return lastNum.substr(1);
+
+    /*
```

Comment:
why is this commented out?

Reply:
Because both if condition's return statements would be doing the same thing, (`return lastNum.substr(1)`) on lines 100 and 104. So we don't need to do the if statements, we know that as long as we have an operator, we need to remove the first one. (If we have one operator (-6) we need to remove it, and if we have two operators in front of the number (--6) we still need to remove the first one. "Operator" here being a term describe the use of an operator (/*+-) which can also serve as the number's sign (-+)).

Reply:
Okay will do!